web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -157,8 +161,30 @@ const useStyles = makeStyles({
 export const DailyViewTable = (props: Props): JSX.Element => {
   const classes = useStyles()
 
+  const stationCodesList: number[] = []
```

Comment:
Can be simplified to:
```
props.dailiesMap.map(daily => daily.code)
```

Reply:
according to Typescript, there is no `map()` function available on the `Map` prototype. Go figure.

Reply:
Sorry it should be:
```
Object.entries(props.dailiesMap).map(daily => daily.code)
```